lblod/frontend-lokaal-mandatenbeheer
# Lmb 599  | Automatic fillin of search form person info

## Description

Make sure that when searching for a person and it did not exist yet the data from the input fields  (first- lastname, rrn) is passed on to the create person form

## How to test

1. create eigen orgaan
2. add mandataris with search of existing person
3. add mandataris with search of NOT existing person
4. Create person see if the fields are prefilled

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/253
